pkg/server/plugin/upstreamauthority/spire/spire_test.go

```diff
@@ -43,6 +43,8 @@ const (
 	"server_port":"_test_data/keys/cert.pem",
 	"server_agent_address":"8090"
 }`
+
+	testTimeout = time.Minute
```

Comment:
is not 1 minute too big? generally this test will take ms locally to at least 1s, things are faked so it will be fast, and in case of error we'll kepp this test taking more tha expected, maybe you can put 10s?

Reply:
I agree, there is some race here that is preventing the test from executing correctly and the ten second timeout is being hit. Increasing it to a minute won't solve the underlying problem.

Reply:
Thank you for the comment. I will close this PR.

Reply:
I appreciate you taking the time to try and address this @ryysud !